Exercise_D and E/assets/style/index.css

```diff
@@ -2,3 +2,121 @@
 This css sheet is linked to practice.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
+.logo {
+    background-color: white;
+    height: 10%;
+}
+
+#logo {
+    height: auto;
+}
+
+.upper {
+    background-image: url('../images/flowershop.jpg');
+    background-size: cover;
+    background-repeat: no-repeat;
+    width: 100%;
+    height: 750px;
```

Comment:
Try to stick to one colour convention when developing.

Also consider using responsive sizes.